### PR TITLE
Default template files.

### PR DESCRIPTION
**Pull request by dschaper**

### CODE REVIEW

File: ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+### Versions
+<!-- Please run `pihole -v` and provide the following: -->
+- Pi-hole:  
+- AdminLTE:  
+- FTL:  
+
+### Platform
+<!-- OS examples: Debian 10, Debian Buster, Ubuntu 18 -->
```
Comment (DL6ER) [inline ISSUE_TEMPLATE/bug_report.md:17]: Maybe write `Ubuntu 18.04` not sure if everyone will get what `Ubuntu 18` is...

File: ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: 
+about: Suggest an idea for this project
```
Comment (DL6ER) [inline ISSUE_TEMPLATE/feature_request.md:3]: Shouldn't this contain the link and a clear indication that opening a FR on Discourse is the right thing to do?

File: ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (dschaper) [reply ISSUE_TEMPLATE/bug_report.md:17]: Okay, that sounds good.

File: ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (dschaper) [reply ISSUE_TEMPLATE/feature_request.md:3]: The "new" way of doing templates uses the `config.yml` file to determine which files and links to show. Filenames don't really mean much other than for organization. The `name:` frontmatter lines are the names of the buttons, so if a file has a blank `name:` then it won't be used/shown. It's just a sample file really in this use. If you check the test repo you can see how it all works in practice.

**Pull request merged**